src/cloudai/workloads/nccl_test/slurm_command_gen_strategy.py

```diff
@@ -29,12 +29,16 @@ def _container_mounts(self, tr: TestRun) -> list[str]:
         mounts: list[str] = []
         env = tr.test.extra_env_vars | self.system.global_env_vars
         if "NCCL_TOPO_FILE" in env:
-            nccl_topo_file = Path(env["NCCL_TOPO_FILE"]).resolve()
+            nccl_topo_file = Path(env["NCCL_TOPO_FILE"]).resolve()  # pyright: ignore [reportArgumentType]
```

Comment:
Same as `env_vars["COMBINE_THRESHOLD"]`, let's try to avoid static checks disabling